README.md

```diff
@@ -783,6 +783,7 @@ Below is an advanced example configuration. You will be able to (optionally) ove
 username: oxidized
 password: S3cr3tx
 model: junos
+#interval in seconds
```

Comment:
You don't need two commits to add this. Also I'd prefer it to be in-line:

interval: 3600  # measured in seconds

I'm curious if 3600 was ambiguous. What other unit makes sense? milliseconds? minutes? hours?